CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 2.8.11)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake/modules")
 
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wno-write-strings -Wall -pthread")
+
+if(DO_NOT_DELAY_TAG_CALC)
```

Comment:
Does this conditional need to be included? The conditional appears in src/CMakeLists.txt and the only references to DO_NOT_DELAY_TAG_CALC are under the src subdirectory.

Or in other words, does something break if you were to remove the entire conditional from here?